Remiworks/Remiworks.Attributes/Initialization/Initializer.cs

```diff
@@ -27,10 +27,33 @@ public Initializer(IServiceProvider serviceProvider)
         public void Initialize(Assembly executingAssembly)
         {
             var types = executingAssembly.GetTypes();
+            CheckDependencies(types);
```

Comment:
Wil je hier uberhaupt wel op ALLE types checken? Daarnaast maakt die van iedere class een instantie aan. Als er classes zijn waarvan het niet de bedoeling is dat deze bij initializatie ingeladen worden omdat deze bijvoorbeeld verbinding maken met een database, kan dit problemen opleveren

Reply:
Daarnaast laat hierdoor het framework een best wel grote footprint achter. Wellicht is het beter om in de event listener en command listener alleen te checken of de geregistreerde classes gecreeerd kunnen worden